nomad/acl_endpoint.go

```diff
@@ -219,32 +219,35 @@ func (a *ACL) GetPolicy(args *structs.ACLPolicySpecificRequest, reply *structs.S
 		return structs.ErrPermissionDenied
 	}
 
-	// If it is not a management token determine if it can get this policy
-	mgt := acl.IsManagement()
-	if !mgt {
-		snap, err := a.srv.fsm.State().Snapshot()
-		if err != nil {
-			return err
-		}
+	// If the policy is the anonymous one, anyone can get it
+	if args.Name != "anonymous" {
+		// If it is not a management token determine if it can get this policy
+		mgt := acl.IsManagement()
+		if !mgt {
```

Comment:
There is a bit too much indentation here.  How about keeping the same level of indentation with something like:

```
	mgt := acl.IsManagement()
	if !mgt && args.Name != "anonymous" {
```

Not your code, but `mgt` is only used once, so I'd inline the call and get rid of variable.

Reply:
Makes sense, thank you!